test/e2e/scrapeconfig_test.go

```diff
@@ -2119,3 +2122,128 @@ var LightSailSDTestCases = []scrapeCRDTestCase{
 		expectedError: true,
 	},
 }
+
+var KumaSDTestCases = []scrapeCRDTestCase{
+	{
+		name: "Server field unspecified",
+		scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
+			KumaSDConfigs: []monitoringv1alpha1.KumaSDConfig{
+				{},
+			},
+		},
+		expectedError: true,
+	},
+	{
+		name: "Server field too short",
+		scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
+			KumaSDConfigs: []monitoringv1alpha1.KumaSDConfig{
+				{
+					Server: "a",
```

Comment:
```suggestion
					Server: "",
```
A string of length 1 is valid.